### PR TITLE
working on cc10

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/stacksAndQueues/queues.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  describe('peek', ()=>{
+    it('11. Can successfully peek into a queue, seeing the expected value', ()=>{
+      let queue = new Queue();
+      queue.enqueue(1);
+      queue.enqueue(2);
+      expect(queue.peek()).toEqual(1)
+    })
+  })
+
+})
```
Comment (adrianhuebner) [inline javascript/code-challenges/stacksAndQueues/queues.test.js:64]: Not all of these queue tests are passing

**Comment by adrianhuebner** [comment]
Would also like to see a README added to this!
